lib/rules/no-unused-placeholders.js

```diff
@@ -86,12 +49,17 @@ module.exports = {
               reportInfo.data && reportInfo.data.type === 'ObjectExpression'
           ) {
             const message = reportInfo.message.value;
+            // https://github.com/eslint/eslint/blob/2874d75ed8decf363006db25aac2d5f8991bd969/lib/linter.js#L986
+            const PLACEHOLDER_MATCHER = /\{\{\s*([^{}]+?)\s*\}\}/g;
+            const set = new Set();
```

Comment:
Nitpick: Can you give this variable a more descriptive name? For example: `placeholdersInMessage`